forge-gui/res/cardsfolder/upcoming/scholar_of_new_horizons.txt

```diff
@@ -0,0 +1,13 @@
+Name:Scholar of New Horizons
+ManaCost:1 W
+Types:Creature Human Scout
+PT:1/1
+K:etbCounter:P1P1:1
+A:AB$ ChangeZone | Cost$ T RemoveAnyCounter<1/Any/Permanent.YouCtrl/a permanent you control> | Origin$ Library | Destination$ Library | ChangeType$ Land.Plains | ChangeNum$ 1 | RememberChanged$ True | Reveal$ True | Shuffle$ False | SubAbility$ ToBattlefieldTapped | SpellDescription$ Search your library for a Plains card and reveal it. If an opponent controls more lands than you, you may put that card onto the battlefield tapped. If you don't put the card onto the battlefield, put it into your hand. Then shuffle.
+SVar:ToBattlefieldTapped:DB$ ChangeZone | ConditionCheckSVar$ Y | ConditionSVarCompare$ GTX | Defined$ Remembered | Optional$ True | Origin$ Library | Destination$ Battlefield | Tapped$ True | Shuffle$ True | SubAbility$ ToHand
```

Comment:
Probably needs a ForgetChanged or something like that?

Make sure the case where you could put it onto the battlefield but choose not to works properly

Reply:
It worked during my testing but will add anyway